multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BindServiceStep.java

```diff
@@ -0,0 +1,81 @@
+package org.cloudfoundry.multiapps.controller.process.steps;
+
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.Map;
+
+import javax.inject.Named;
+
+import org.cloudfoundry.client.lib.ApplicationServicesUpdateCallback;
+import org.cloudfoundry.client.lib.CloudControllerClient;
+import org.cloudfoundry.client.lib.CloudOperationException;
+import org.cloudfoundry.multiapps.common.SLException;
+import org.cloudfoundry.multiapps.controller.client.lib.domain.CloudApplicationExtended;
+import org.cloudfoundry.multiapps.controller.client.lib.domain.CloudServiceInstanceExtended;
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.cloudfoundry.multiapps.controller.process.variables.Variables;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+
+@Named("bindServiceStep")
```

Comment:
You could rename to bindServiceToAppStep becaus we could support route services in future.